src/main/java/org/example/Main.java

```diff
@@ -19,10 +21,31 @@ public static void main(String[] args) {
             out.println(input);
             if (CORRECT_NUMBER.equals(input)) {
                 out.println("Wszedles!");
+                menu();
                 return;
             }
             numberOfTries++;
             out.println("Nie prawidlowe haslo!\n");
         }
     }
+    public static void menu() {
+
+        Scanner number = new Scanner(in);
```

Comment:
Why do we create new Scanner istance here and why it's called number? Inject it.